app/views/layouts/mailer.html.erb

```diff
@@ -53,25 +57,23 @@
       }
 
       #layout__footer {
-          padding: 1rem;
           font-size: 0.75rem;
       }
 
       .layout__separator {
           border-top: 1px dashed lightgray;
-          margin: 1rem 0;
       }
 
   </style>
-  <title>High School Hackathons</title>
+  <title>hackathons.hackclub.com</title>
 </head>
 
 <body>
 <div id="layout">
   <header id="layout__header">
-    <%= image_tag "flag-orpheus-top.png", id: "layout__logo" %>
+    <%= image_tag "flag-standalone", id: "layout__logo" %>
```

Comment:
I love the new design! mobile's just looking a little funky

![image](https://github.com/hackclub/hackathons-backend/assets/20099646/d3ffe873-6c55-4026-bc53-44f44d0bee55)
http://localhost:3000/rails/mailers/hackathons/digest_mailer/digest.html?locale=en